src/models/messages.js

```diff
@@ -0,0 +1,26 @@
+import { Room } from './room.js';
+import DataTypes from 'sequelize';
```

Comment:
The `DataTypes` should be imported using destructuring: `import { DataTypes } from 'sequelize';` to correctly access the data types.